.github/workflows/verifiers.yml

```diff
@@ -193,21 +193,25 @@ jobs:
         env:
           cache-name: cache-knife
         with:
-          path: ~/.cargo/bin/cache-knife
+          path: metamath-knife/target/release/metamath-knife
           key: ${{ runner.os }}-build-${{ env.cache-name }}
       - name: Install rust
         if: ${{ !steps.cache-metamath-knife.outputs.cache-hit }}
         uses: hecrj/setup-rust-action@v1
         with:
           rust-version: stable
-      - run: git clone --depth 1 https://github.com/david-a-wheeler/metamath-knife.git
-      - run: cd metamath-knife && cargo build --release
+      - name: Install metamath-knife
+        if: ${{ !steps.cache-knife.outputs.cache-hit }}
+        run: |
+          git clone --depth 1 https://github.com/david-a-wheeler/metamath-knife.git &&
```

Comment:
The repository has moved recently, now it is:
```
https://github.com/metamath/metamath-knife.git
```

Reply:
Ah, nice. I was hoping we'd get around to that move.

Reply:
Changed.